CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# How to become a contributor and submit your own code
+
+We'd love to accept your issues and (especially) patches! If you have found a
+bug you can submit an issue describing it and one of the repo maintainers will
+get back to you. Alternately you can submit a PR and the sections below outline
+the process.
+
+## Contributing a Patch
+
+1. Fork the desired repo, develop and test your code changes.
+1. Ensure that your code has an appropriate set of unit tests which all pass.
+1. Break your changes in to a series of logical self-contained commits.
+1. Include release notes files when appropriate following the guidelines
+   in Neptune `Contributing to Neptune-Versioned Libraries <https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-contributor.rst>`_
+1. Submit your pull request.
+
+## Reviewing and Rebasing and Merging Patches
+
+1. All feedback should be professionally given and non-personal. Offensive,
+   intimidating, insulting, racist, homophobic, and sexist comments and
+   behavior will not be tolerated.
```

Comment:
I am not sure if github themselves have something also covering this we should link to.

Reply:
I remember hearing about code of conduct.

Reply:
https://help.github.com/articles/adding-a-code-of-conduct-to-your-project/